config/capsule-answers.yaml

```diff
@@ -0,0 +1,5 @@
+---
+certs:
+  generate: false
+  deploy: true
+capsule: true
```

Comment:
Note: there is an existing issue related to having puppet disabled : http://projects.theforeman.org/issues/11197 . I am surprised that more folks have not hit it.